packages/react-native/React/Views/RCTTransformOrigin.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <Foundation/Foundation.h>
+#import <yoga/Yoga.h>
```

Comment:
Why is this pulling in Yoga? Does this just want a struct that can be a percentage or integral number? What would `YGValueAuto` or `YGValueUndefined` mean here?

Reply:
Would probably be better to avoid the dependency, since this system does not interact with Yoga.

Apart from general goodness of a clean dependency graph, this currently breaks the internal Buck build because this target didn't need to preciously depend on Yoga.

Reply:
It looks like we missed tackling this before merging this code. Would be great to clean up.